charts/govuk-rails-app/values.yaml

```diff
@@ -11,21 +11,17 @@ externalSecrets:
 
 ingress:
   enabled: false
-  name:
-  host: ""
-  path: "/"
-  pathType: Prefix
   annotations:
     alb.ingress.kubernetes.io/scheme: internet-facing
     alb.ingress.kubernetes.io/target-type: ip
     alb.ingress.kubernetes.io/listen-ports: '[{"HTTP": 80}, {"HTTPS": 443}]'
     alb.ingress.kubernetes.io/ssl-redirect: "443"
     alb.ingress.kubernetes.io/healthcheck-path: /healthcheck/ready
+    alb.ingress.kubernetes.io/load-balancer-name:
   hosts:
-  - host:
-    paths:
-    - path: /
```

Comment:
Will need to add Multiple path handling in a further iteration of the chart

Reply:
Ah that's my bad for templating the Ingress values 😅 Do we have any cases where we need multiple paths though? I haven't come across any yet.

Reply:
I have not seen the need as of yet - but if we do need it it will be easily to implement without breaking anything - i will test this when i add Whitehall.